Add Best Practices and Example Apps Links (#528)
smartdevicelink/sdl_app_library_guides: docs/Getting Started/Example Apps/index.md

The [JavaScript Suite repository on GitHub](https://github.com/smartdevicelink/sdl_javascript_suite/tree/master/examples) provides example apps for both the browser and for NodeJS. This includes a WebEngine app, a WebSocket client app, a WebSocket server app, and a TCP client app. The examples in the folders already come with their own SDL library build files. Check each example app's `readme.md` file for more information on how to run the respective app.
!@

## Additional Examples

> For more examples go to the [Smart Device Link Examples](https://github.com/SmartDeviceLink-Examples) GitHub organization. Download or clone any of these projects.

Reviewer: ```suggestion
For more examples go to the [SmartDeviceLink Examples](https://github.com/SmartDeviceLink-Examples) GitHub organization. Download or clone any of these projects.
```

@![iOS]

> For iOS, the examples available include the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios), and the [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_ios). The [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios) uses the OpenWeather API to implement a basic connected weather app with SDL UI. This example showcases changing screen template items for certain weather forecasts, displaying hourly and daily weather in popup menus, and showing weather alerts with SDL Alerts. Another example, the [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_ios) utilizes the MapBox API to create a basic navigation app. The example navigation app can be used as a reference for developers who want to create their own navigation app.

Reviewer: ```suggestion
The examples available include an [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios) and an [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_ios). 

The [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_ios) uses the OpenWeather API to implement a basic connected weather app with SDL UI. This example showcases changing screen template items for certain weather forecasts, displaying hourly and daily weather in popup menus, and showing weather alerts with SDL Alerts. 

The [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_ios) utilizes the MapBox API to create a basic video streaming map app. The example navigation app can be used as a reference for developers who want to create their own navigation app.
```

!@

@![android]

> For android, the examples available include the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android), [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android), and the [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android). The [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android) uses the OpenWeather API to implement a basic connected weather app with SDL UI. This example showcases changing screen template items for certain weather forecasts, displaying hourly and daily weather in popup menus, and showing weather alerts with SDL Alerts. In addition, the [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android) utlizes the MapBox API to create a basic navigation app. The example navigation app can be used as a reference for developers who want to create their own navigation app. In addition, the [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android) showcases display capabilities of SDL.

Reviewer: ```suggestion
The examples available include the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android), [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android), and the [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android). 

The [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android) uses the OpenWeather API to implement a basic connected weather app with SDL UI. This example showcases multiple screens for different weather forecast types, displaying hourly and daily weather in popup menus, and showing weather alerts with SDL Alerts. 

The [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android) uses the MapBox API to create a basic video streaming map app. The example navigation app can be used as a reference for developers who want to create their own navigation app.

The [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android) showcases various capabilities of SDL.
```

});
```
!@

> 
> ## Additional Examples
> For more examples go to the [SmartDeviceLink Examples](https://github.com/SmartDeviceLink-Examples) GitHub organization. Download or clone any of these projects.
> 

Reviewer: ```suggestion

@![android, iOS]
## Additional Examples
For more examples go to the [SmartDeviceLink Examples](https://github.com/SmartDeviceLink-Examples) GitHub organization. Download or clone any of these projects.
!@

```

!@

@![android]
The examples available include the [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android), [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android), and the [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android). 

The [example weather app](https://github.com/SmartDeviceLink-Examples/example_weather_app_android) uses the OpenWeather API to implement a basic connected weather app with SDL UI. This example showcases multiple screens for different weather forecast types, displaying hourly and daily weather in popup menus, and showing weather alerts with SDL Alerts. 

The [example navigation app](https://github.com/SmartDeviceLink-Examples/example_navigation_app_android) uses the MapBox API to create a basic video streaming map app. The example navigation app can be used as a reference for developers who want to create their own navigation app.

The [SDL Capabilities app](https://github.com/SmartDeviceLink-Examples/SDL-Capabilities-Android) showcases various capabilities of SDL.
!@

> !!! NOTE
> Some examples require obtaining API tokens from third parties for data and services. For all of thse examples follow the setup instructions as outlined in their **README.md**.
> !!!

Reviewer: ```suggestion
@![android, iOS]
!!! NOTE
Some examples require obtaining API tokens from third parties for data and services. For all of thse examples follow the setup instructions as outlined in their **README.md**.
!!!
!@
```
